panel/entry_points.py

```diff
@@ -0,0 +1,51 @@
+""" This module defines a function for iterating over entry points
```

Comment:
The root directory of the panel package is getting a little crowded. I think it's time to bite the bullet and add a subpackage for some stuff, not quite sure what to sensibly group in there yet though. I'll consider and get back to you.